### ISSUE TITLE
Documentation for usage with Karma/Angular 4+

### ISSUE THREAD

**Issue by kbirger** [mochajs/mocha#3717]
I assume that since older versions of mocha worked with older versions of Angular, that this is still possible, but all of the references I can find online for this are varied and not helpful. I think it would be helpful for this project if there was an official documentation for how to set up mocha for Karma and Angular.

**Comment by boneskull** [comment]
the wiki would make sense for this... but we'd need someone who's familiar with Angular to create it.

**Comment by kbirger** [comment]
@weswhite I'm not an owner of this project, but at a glance, this seems like a reasonable example to base it on: 
https://github.com/shellscape/mocha-chrome

I greatly appreciate it by the way.

**Comment by boneskull** [comment]
@weswhite yes, whatever is helpful for @kbirger would likely be helpful for the greater community.  I can't say what's needed because I don't use Mocha with Karma & Angular (though Mocha uses Karma to test itself)

**Comment by weswhite** [comment]
@boneskull I figured out how to do this. Where should I put the steps? First open source contribution, sorry...

**Comment by boneskull** [comment]
@weswhite 	Create a new [wiki page](https://github.com/mochajs/mocha/wiki).  make sure you link to it from the list of tutorials on the front wiki page.  thanks for your help!! :heart:

**Comment by oscarr-reyes** [comment]
how is this going?.. i would really love to see how to make angular tests using mocha
